### ISSUE TITLE
Allow to disable, debug telemetry

### ISSUE THREAD

**Issue by adietish** [adietish/intellij-redhat-telemetry#3]
For UI tests we have to be able to disable telemetry
For debugging purposes, we need to use a different segment account (aka segment writeKey).
The idea is to have a mode where
* disabled = no telemetry
* debug = telemetry is reported to separate segment account

**Comment by jeffmaury** [comment]
Can't we use a system property instead of **~/.redhat/com.redhat.devtools.intellij.telemetry** to pass the mode ?

**Comment by adietish** [comment]
I was thinking of this file bcs we also need a storage for enablement (we wanted to use a file in ~/.redhat/ instead of the intellij preferences so that enablement would be global for all intellij installations that may exist). 
We could use both where system properties would override the value in file. Makes sense?

**Comment by jeffmaury** [comment]
Yes because I don't see the rationale for the local file unless I missed something

**Comment by adietish** [comment]
We allow users to enable/disabled via settings. We discussed storing this to some file in ~/.redhat instead of IJ preferences so that it would be valid for all IJ instances that exist on the same machine. Am I misunderstanding something?

**Comment by jeffmaury** [comment]
Yeah I forgot that disabled state is stored in ~/.redhat. LGTM
